bitnami/postgresql/templates/svc-headless.yaml

```diff
@@ -1,12 +1,13 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "common.names.fullname" . }}-headless
-  labels:
-  {{- include "common.labels.standard" . | nindent 4 }}
-  {{- if .Values.commonLabels }}
-  {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-  {{- end }}
+  name: {{ printf "%s-hl" (include "postgresql.primary.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Comment:
This sounds like something to be added to a helper, as it is used in other parts of the yamls

Reply:
I'm against adding named templates without if/else logic.. I mean they don't add any value and they're just a wrapper for a template that can be written in a single line.

Reply:
There are 4 operations in the line actually. 

 - Build a string with a suffix
 - Call the fullname
 - Truncate to 63 characters
 - Trim the suffix

In case of changing this, it's highly likely that someone may forget that it is present in more places. Plus, if a user wants to use the headless service in a custom, rendered configuration, they will have to use all those operations.

Reply:
All right